common/deliver/deliver.go

```diff
@@ -313,7 +313,11 @@ func (h *Handler) deliverBlocks(ctx context.Context, srv *Server, envelope *cb.E
 		}
 
 		if status != cb.Status_SUCCESS {
-			logger.Errorf("[channel: %s] Error reading from channel, cause was: %v", chdr.ChannelId, status)
+			if status == cb.Status_NOT_FOUND {
+				logger.Errorf("[channel: %s] Error reading from channel, block [%d] not found.", chdr.ChannelId, block.Header.Number)
```

Comment:
these log entries should be warnings, not errors.

Reply:
Updated the commit.